File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/sync/SyncView.java
package io.muun.apollo.presentation.ui.fragments.sync;


import io.muun.apollo.presentation.ui.base.SingleFragmentView;

public interface SyncView extends SingleFragmentView {

    String ARG_FROM_STEP = "from_step";

    void setLoading(boolean isLoading);

    void setIsExistingUser(boolean isExistingUser);

    void setUpPinCode(boolean canCancel);
}

File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/sync/SyncView.kt
package io.muun.apollo.presentation.ui.fragments.sync

import io.muun.apollo.presentation.ui.base.SingleFragmentView

interface SyncView : SingleFragmentView {

    override fun setLoading(isLoading: Boolean)

    fun setIsExistingUser(isExistingUser: Boolean)

    fun setUpPinCode(canCancel: Boolean)
}